src/sdk-reference/js/6/kuzzle-event-emitter/prependListener/snippets/prepend-listener.test.yml

```
@@ -0,0 +1,9 @@
+name: kuzzleEventEmitter#prepend-listener
+description: Prepends a new listener for the given event
+hooks:
+  before:
+  after:
+template: eventemitter
+expected: Successfully prepended 1 new listener
```

If my previous remark is applied, then this line should be changed for the following:

```suggestion
expected:
  - ^listener2$
  - ^listener1$
```